### PR TITLE
pecunia 2.1.7 (new cask)

### PR DESCRIPTION
**Pull request by clashman**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by krehel** [comment]
@reitermarkus - do you mind providing an opinion here?  This seems like a very specific software, e.g. is it it only useful for the German market?

**Comment by reitermarkus** [comment]
HBCI seems to be a German thing, but it seems to support PSD2, which is an EU thing, since version 2.0.

File: Casks/p/pecunia.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  sha256 "2e9822d38b6db4b37e7a561c11c3f6e87a6bbd7a2c2bd8fbaa6eedacca55f0ee"
+
+  url "https://pecuniabanking.de/downloads/#{version}/Pecunia.zip"
+  name "Pecunia"
+  desc "Online banking app with support for HBCI"
+  homepage "https://pecuniabanking.de/"
+
+  livecheck do
+    url "https://pecuniabanking.de/index.php/downloads"
+    regex(%r{href="/downloads/([\d.]+)/Pecunia.zip}i)
```
Comment (bevanjkay) [inline Casks/p/pecunia.rb:12]:
```suggestion
    regex(%r{href=.*?downloads/v?(\d+(?:\.\d+)+)/Pecunia\.zip}i)

**Comment by clashman** [comment]
> @reitermarkus - do you mind providing an opinion here? This seems like a very specific software, e.g. is it only useful for the German market?

> HBCI seems to be a German thing, but it seems to support PSD2, which is an EU thing, since version 2.0.

Why is any of this important for inclusion into the repo?

**Comment by bevanjkay** [comment]
Casks that are included here (`homebrew-cask`) should meet a certain level of notability/demand. If they are only used by a couple of individuals, then they are best hosted in a personal tap. The questions are to ascertain the popularity/notability of the application.

**Review by p-linnane — APPROVED** [review]

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @clashman.

**Pull request merged**